test_resources/instance_analytics_skip/collections/tYhkn_-pFTWcCTSwhgCBP_instance_analytics/tYhkn_-pFTWcCTSwhgCBP_instance_analytics.yaml

```diff
@@ -2,7 +2,6 @@ authority_level: null
 description: null
 archived: false
 slug: instance_analytics
-color: '#509EE3'
```

Comment:
Fixes this failed test
https://github.com/metabase/metabase/actions/runs/6356184320/job/17265433627?pr=34239#step:4:417

cc @bshepherdson